clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -691,7 +691,7 @@ private void destroyServerStore(ClientDescriptor clientDescriptor, DestroyServer
 
     final Set<ClientDescriptor> clients = storeClientMap.get(name);
     if (clients != null && !clients.isEmpty()) {
-      throw new ResourceBusyException("Can not destroy clustered tier '" + name + "': in use by " + clients.size() + " clients");
+      throw new ResourceBusyException("Cannot destroy clustered tier '" + name + "': in use by " + clients.size() + " other client(s)");
```

Comment:
Should it?  I'm just thinking that if you see this exception: `Cannot destroy clustered tier 'foo': in use by 1 other client(s)` what can you do about it?  Seeing `Cannot destroy clustered tier 'foo': in use by Client[192.168.0.42:56789]` is much more actionable.